utils/firestore.js

```diff
@@ -431,6 +431,64 @@ const retrieveParticipantsEligibleForIncentives = async (siteCode, roundType, is
     }
 }
 
+const retrieveParticipantsDataDestruction = async (totalDocNumber = 10) => {
```

Comment:
what is the purpose of `totalDocNumber` parameter? it doesn't look like we will ever pass it in, so just set a variable within the function

Reply:
The purpose of `totalDocNumber` is to allow us to adjust totalDocNumber without having to edit the contents of the function. e.g. In the future we can change it to `retrieveParticipantsDataDestruction(20)` `retrieveParticipantsDataDestruction(30)`